<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graph_simulationClass</class>
 <widget class="QMainWindow" name="graph_simulationClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>graph_simulation</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>10</y>
      <width>281</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Stream Data Load Setting</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>203</width>
       <height>30</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="radioBtn_streamOn">
        <property name="text">
         <string>On</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_streamOff">
        <property name="text">
         <string>Off</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushBtn_streamConfirm">
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>80</y>
      <width>281</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Data Loading Status</string>
    </property>
    <widget class="QTextBrowser" name="label_dataStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>631</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabbox" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>901</width>
      <height>741</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Data_File"/>
   </widget>
   <widget class="QMenu" name="menuFunctions">
    <property name="title">
     <string>Functions</string>
    </property>
    <widget class="QMenu" name="menuSearch_Author">
     <property name="title">
      <string>Search Author</string>
     </property>
     <addaction name="actionSearch_one_Author"/>
     <addaction name="actionSearch_two_Authors"/>
    </widget>
    <addaction name="actionTop_10_Authors_colored_graph"/>
    <addaction name="actionAuthor_filtering_animation"/>
    <addaction name="actionRecommend"/>
    <addaction name="menuSearch_Author"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFunctions"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad_Data_File">
   <property name="text">
    <string>Load Data File</string>
   </property>
  </action>
  <action name="actionTop_10_Authors_colored_graph">
   <property name="text">
    <string>Top-10 Authors colored graph</string>
   </property>
  </action>
  <action name="actionAuthor_filtering_animation">
   <property name="text">
    <string>Author filtering animation</string>
   </property>
  </action>
  <action name="actionSearch_one_Author">
   <property name="text">
    <string>Search one Author</string>
   </property>
  </action>
  <action name="actionSearch_two_Authors">
   <property name="text">
    <string>Search two Authors</string>
   </property>
  </action>
  <action name="actionRecommend">
   <property name="text">
    <string>Recommend</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="graph_simulation.qrc"/>
 </resources>
 <connections/>
</ui>
